src/vm/clrvarargs.cpp

```diff
@@ -33,10 +33,14 @@ void VARARGS::MarshalToManagedVaList(va_list va, VARARGS *dataout)
 {
     WRAPPER_NO_CONTRACT
 
+#if !(defined(_TARGET_ARM_) && defined(PLATFORM_UNIX))
```

Comment:
Just ifdef out for FEATURE_CORECLR then or is there another more fitting define to guard it? The functions are referenced from stubhelper.cpp so if we could ifdef out the references there in the FCall methods we could remove this file from the build entirely I think.